src/v2/view-builder/ViewFactory.js

```diff
@@ -60,7 +60,7 @@ const VIEWS_MAPPING = {
   },
   'enroll-factor': {
     email: RequiredFactorEmailView, // TODO EnrollFactorEmailView is unimplemented
-    password: EnrollFactorPasswordView,
+    password: EnrollAuthenticatorPasswordView,
```

Comment:
do not change anything `factor` related. just keep there and we will clean up once for all (probably post-m1)